blocks/images-grid/images-grid.js

```diff
@@ -0,0 +1,51 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
```

Comment:
Can you use ```createElement ``` from script.js for creating elements. You can also pass classes to it.